### ISSUE TITLE
clicking on an openlayers canvas doesn't trigger any events

### ISSUE THREAD

**Issue by BasieP** [cypress-io/cypress#1016]
Hi,

When i have a test like this:

```js
it('should draw little circle on a map', function () {
	cy.visit('http://openlayers.org/en/latest/examples/draw-features.html');
	cy.get('canvas').click();
	cy.get('canvas').click(100, 100);
});
```

it doesn't trigger any events (the openlayers example should draw dots on the map)
Perhaps this is because the mousemove and mouseover events are not triggered... i don't know the inner workings of openlayers on this... (although i do know they do stuff a little different)

the problem we now have i can't test any geo applications based on openlayers (which are most of them)

**Comment by brian-mann** [comment]
The simplest thing for you to do here is simply fire the underlying events you want / care about

`cy.get('canvas').trigger('someEvent', props)`

You'll have to understand what the canvas is wired up to. You can use the dev tools to see which DOM event listeners its bound to.

**Comment by Hersir88** [comment]
Any news here? As I want to test my canvas based app and I need to trigger clicks on it bit right now, no clicks are registered on canvas

**Comment by BasieP** [comment]
Canvas is not a problem but Openlayers is..
Workaround is to use .trigger() mousedown  and mouseup manually.

**Comment by Renaud009** [comment]
```
cy.visit('http://openlayers.org/en/latest/examples/draw-features.html')
cy.get('canvas')
            .trigger('pointerdown')
            .trigger('pointerup')
```

**Comment by tascioglu** [comment]
@Renaud009 any ideas..?

**Comment by Bkucera** [comment]
Currently we do not fire pointerdown or pointerup events during cy.click(), this will be fixed with #2956
